2020-12-01 13:44:15,325 INFO    MainThread:3062 [wandb_setup.py:_flush():70] setting env: {}
2020-12-01 13:44:15,325 INFO    MainThread:3062 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2020-12-01 13:44:15,325 INFO    MainThread:3062 [wandb_setup.py:_flush():70] setting login settings: {}
2020-12-01 13:44:15,325 INFO    MainThread:3062 [wandb_setup.py:_flush():70] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fe2ac73a940>
2020-12-01 13:44:15,325 INFO    MainThread:3062 [wandb_init.py:_log_setup():303] Logging user logs to /home/namitrak/Documents/Machine Learning/Image segmentation/wandb/run-20201201_134415-320kjawx/logs/debug.log
2020-12-01 13:44:15,325 INFO    MainThread:3062 [wandb_init.py:_log_setup():304] Logging internal logs to /home/namitrak/Documents/Machine Learning/Image segmentation/wandb/run-20201201_134415-320kjawx/logs/debug-internal.log
2020-12-01 13:44:15,967 INFO    MainThread:3062 [wandb_run.py:_console_start():1218] atexit reg
2020-12-01 13:44:15,968 INFO    MainThread:3062 [wandb_run.py:_redirect():1085] redirect: SettingsConsole.WRAP
2020-12-01 13:44:15,968 INFO    MainThread:3062 [wandb_run.py:_redirect():1116] Wrapping output streams.
2020-12-01 13:44:15,968 INFO    MainThread:3062 [wandb_run.py:_redirect():1132] Redirects installed.
2020-12-01 13:44:18,992 INFO    MainThread:3062 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:44:18,996 INFO    MainThread:3062 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:44:41,388 INFO    MainThread:3062 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:44:41,392 INFO    MainThread:3062 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:44:41,520 INFO    MainThread:3062 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:44:41,526 INFO    MainThread:3062 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:44:41,526 INFO    MainThread:3062 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:44:41,529 INFO    MainThread:3062 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:44:41,748 INFO    MainThread:3062 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:44:41,751 INFO    MainThread:3062 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:44:44,176 INFO    MainThread:3062 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:44:44,182 INFO    MainThread:3062 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:44:44,184 INFO    MainThread:3062 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:44:44,189 INFO    MainThread:3062 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:44:44,190 INFO    MainThread:3062 [wandb_watch.py:watch():35] Watching
2020-12-01 13:44:45,792 INFO    MainThread:3062 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:44:54,848 INFO    MainThread:3062 [wandb_run.py:_atexit_cleanup():1188] got exitcode: 0
2020-12-01 13:44:54,848 INFO    MainThread:3062 [wandb_run.py:_restore():1160] restore
2020-12-01 13:44:54,851 INFO    MainThread:3062 [wandb_run.py:_wait_for_finish():1305] got exit ret: uuid: "3a065f97aa1e44a7bc5a6f84ccd1a446"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
      media_count: 1
    }
    pusher_stats {
      uploaded_bytes: 10064
      total_bytes: 10064
    }
  }
}

2020-12-01 13:44:56,855 INFO    MainThread:3062 [wandb_run.py:_wait_for_finish():1305] got exit ret: uuid: "a59ff291bbf74c9696a303a44d377f97"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
      media_count: 1
    }
    pusher_stats {
      uploaded_bytes: 13540
      total_bytes: 13540
    }
  }
}

2020-12-01 13:44:57,907 INFO    MainThread:3062 [wandb_run.py:_show_summary():1450] rendering summary
2020-12-01 13:44:57,908 INFO    MainThread:3062 [wandb_run.py:_show_files():1517] logging synced files
2020-12-01 13:44:57,976 INFO    MainThread:3062 [internal.py:handle_exit():138] Internal process exited
